package.json

```diff
@@ -6,7 +6,8 @@
   "main": "index.ts",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "bidi-server": "npm run build && node ./src/.build/index.js",
+    "bidi-server": "npm run build && npm run server --",
```

Comment:
What does the trailing `--` do here? Unless it avoid the need for `--` between `npm run bidi-server` and arguments I suspect it doesn't do anything :)

Reply:
trailing `--` is needed to pass the cli arguments inside the script. otherwise they will be interpreted as an NPM params.
https://docs.npmjs.com/cli/v7/commands/npm-run-script